site.yml

```diff
@@ -9,9 +9,9 @@ content:
     - HEAD
   - url: https://github.com/owncloud/docs.git
     branches:
+    - '10.4'
     - '10.3'
     - '10.2'
-    - '10.1'
```

Comment:
This should be kept, otherwise it also won't appear on the version list anymore and we got more reasons why we can't wipe old files from the deployment bucket.

Reply:
Fair enough.